pkgs/os-specific/linux/metastore/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, libbsd, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "1.1.2";
+  pname = "metastore";
+  src = fetchFromGitHub {
+    owner = "przemoc";
+    repo = "metastore";
+    rev = "v${version}";
+    sha256 = "0mb10wfckswqgi0bq25ncgabnd3iwj7s7hhg3wpcyfgckdynwizv";
+  };
+
+  buildInputs = [ libbsd ];
+  installFlags = [ "DESTDIR=$(out)" "PREFIX=/" ];
+
+  meta = {
+    description = "Store and restore metadata from a filesystem";
+    homepage = "https://software.przemoc.net/#metastore";
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = stdenv.lib.maintainers.sstef;
```

needs to be an array
```suggestion
    maintainers = with maintainers; [ sstef ];
```